src/pages/documentation/mobile-core/lifecycle/tabs/api-reference.md

```diff
@@ -61,10 +77,24 @@ contextData.put("myapp.category", "Game");
 MobileCore.lifecycleStart(additionalContextData);
 ```
 
-<Variant platform="ios" api="lifecycle-start" repeat="9"/>
+<Variant platform="ios" api="lifecycle-start" repeat="13"/>
```

Comment:
This one should be 14, I think? Honestly, the easiest way to test is to run it locally and see, unfortunately.